test/plugins/gcc/test-writer/should-create-test-index-with-test-dir/test/exclude-a.test.js

```diff
@@ -0,0 +1,3 @@
+goog.require('dep1');
```

Comment:
Point of confusion for me: is there a reason these files are named "exclude" when they are being expected to be included in the test results? Are there files that we would ever expect to be excluded?

Reply:
We currently run any `*.test.js` file through Mocha, via the glob pattern in the `test:run` script. I added a test to verify the webpack test entry file is created correctly, and that code finds `*.test.js` files in a project's test directory. The test resources for this were picked up by Mocha because they matched the include glob, and it tried executing them which is both unwanted and failed because they aren't actually resolver tests.

To fix this, I changed the include glob to `test/**/!(exclude)*.test.js`, which ignores `exclude*.test.js` files in the runner. This was based on the suggested solution in mochajs/mocha/issues/1577.